src/helpers/+_sign.ts

```diff
@@ -0,0 +1,32 @@
+import type { PcbSilkscreenPath } from "@tscircuit/soup"
```

Comment:
Can we make this filename a bit safer for different operating systems, we could call it plus_sign